tests/unit/test_crud.py

```diff
@@ -646,22 +633,9 @@ def test_get_stripe_customer_by_nonexistent_fxa_id(
     assert customer is None
 
 
-@pytest.fixture()
-def stripe_objects(
-    dbsession,
-    example_contact,
-    maximal_contact,
-    stripe_price_data,
-    stripe_subscription_data,
-    stripe_subscription_item_data,
-    stripe_invoice_data,
-    stripe_invoice_line_item_data,
-):
-    """
-    Create two complete trees of Stripe objects.
-
-    We don't use ForeignKeys for Stripe relations, because the object may come
-    out of order for foreign key contraints. This helps check that the manually
+class TestStripeRelations:
```

Comment:
I see that `self` isn't used anywhere. 
Did you create a class to "group" the tests? Shall we move them to `test_crud_stripe_releations` instead?

Reply:
I did, both to group the tests and so that I could define a fixture that's autoused _only_ in the scope of these tests.

> Shall we move them to `test_crud_stripe_releations` instead

Do you mean a new module i.e. `test_crud_stripe_releations.py`?